bundle/manifests/aws-load-balancer-operator.clusterserviceversion.yaml

```diff
@@ -274,11 +275,12 @@ spec:
                 - --image=$(RELATED_IMAGE_CONTROLLER)
                 - --namespace=$(TARGET_NAMESPACE)
                 - --trusted-ca-configmap=$(TRUSTED_CA_CONFIGMAP_NAME)
+                - --webhook-disable-http2
                 command:
                 - /manager
                 env:
                 - name: RELATED_IMAGE_CONTROLLER
-                  value: quay.io/aws-load-balancer-operator/aws-load-balancer-controller@sha256:f359f63169432cf8029b189f6605f21d660f0f805b91d55fe964185d0b8ae0b2
+                  value: quay.io/aws-load-balancer-operator/aws-load-balancer-controller@sha256:d8b5e9a91aca2a4a4de7f9bd2b614c5ba3d4cc62fa8967e94e9539fd7c1940a9
```

Comment:
For reference, here's the corresponding manifest: https://quay.io/repository/aws-load-balancer-operator/aws-load-balancer-controller/manifest/sha256:d8b5e9a91aca2a4a4de7f9bd2b614c5ba3d4cc62fa8967e94e9539fd7c1940a9

The manifest labels reference the expected aws-load-balancer-controller commit: `"vcs-ref"="0ae6e1a96d44f67fe852e91ec31a79d138022cd2" "vcs-type"="git" "vcs-url"="https://github.com/openshift/aws-load-balancer-controller"`, which is <https://github.com/openshift/aws-load-balancer-controller/commit/0ae6e1a96d44f67fe852e91ec31a79d138022cd2>.

Reply:
Comments added to the manager manifest.